laser_slam/include/laser_slam/laser_track.hpp

```diff
@@ -47,6 +47,9 @@ class LaserTrack {
   /// \brief Get one local cloud in world frame.
   void getLocalCloudInWorldFrame(const Time& timestamp, DataPoints* out_point_cloud) const;
 
+  /// \brief Get all laser scans.
+  const std::vector<LaserScan>* getLaserScans() const;
```

Comment:
I see! I am not sure which is the best option here. I believe @HannesSommer would be in favor of the const ref, right?